src/editors/components/EditorFooter/index.jsx

```diff
@@ -18,8 +19,15 @@ import { selectors, thunkActions } from '../../data/redux';
 import messages from './messages';
 import * as module from '.';
 
-export const handleSaveClicked = (props) => () => saveBlock(props);
-export const handleCancelClicked = ({ returnUrl }) => navigateCallback(returnUrl);
+export const handleSaveClicked = (props) => () => {
+  sendTrackEvent('edx.ui.authoring.editor.save', {});
```

Comment:
Data bits that would be nice to send along with this event (and the one below):
- Block ID
- Block type
- Course run ID (might be unavailable)

Reviewers, how would I access this info to send it to the handlers for sending to Segment?

Reply:
You could have that information passed in from the redux state using mapStateToProps.

Reply:
Does this make sense?

Reply:
I believe we should create some sort of returnAnalytics selector in /data/redux/app/selectors.js to get these values.

The sendTrackEvent call should be placed in hooks.js since this is where we've been writing our logics. We should try to keep the EditorFooter component as 'dumb' as possible.

Reply:
@connorhaugh @rayzhou-bit I've attempted to incorporate your feedback - but I've been unable to make a commit that passes tests, mostly because of my lack of Redux knowledge. This task will be completed much more quickly if one of you takes over this PR and makes the needed changes - so I recommend you talk amongst yourselves and decide who will complete it. Please let me know your decision - thanks.